src/components/PatientProfile/PatientProfilePage.tsx

```diff
@@ -341,7 +353,11 @@ const PatientProfilePage = ({
         borderBottom={`1px solid ${Colours.BorderLightGray}`}
       >
         <Typography variant="h4">
-          {mode === 'new' ? 'Add a patient' : 'Edit patient'}
+          {mode === 'new'
+            ? 'Add a patient'
+            : isRestore
+            ? 'Restore patient'
+            : 'Edit patient'}
```

Comment:
nit: i really don't like how unreadable this is, it's really bad because it's very confusing until you look at it 3 or 4 times; that being said, it's very obvious once you figure it out so i'm on the fence on changing it lol

one option is to take the whole thing and put it in a separate `const` and/or add some ifs or brackets to make it a little clearer

Reply:
I'll abstract out to increase readability